runtime/tests/interpreter/account_test.go

```diff
@@ -997,2300 +995,109 @@ func TestInterpretAuthAccount_borrow(t *testing.T) {
 	})
 }
 
-func TestInterpretAuthAccount_link(t *testing.T) {
```

Comment:
Diff looks wrong, because this and the next text (unlink) got removed, shifting the tests below up. 
Maybe view in split mode instead